packages/shared/src/hooks/squads/useSourceModerationList.ts

```diff
@@ -101,6 +104,9 @@ export const useSourceModerationList = ({
       }),
     onSuccess: () => {
       displayToast('Post(s) approved successfully');
+      logEvent({
+        event_name: LogEvent.ApprovePost,
+      });
```

Comment:
We should use postLogEvent for these, if you check taxonomy it states this under `Post events` it's like a default of items that we add.

Let me know if this is unclear or you want to know what I mean :) 

![Screenshot 2024-11-15 at 10 50 29](https://github.com/user-attachments/assets/4a635f1c-5625-4e64-be97-933623fc8719)

Reply:
@rebelchris  but at this point we don't have a full post object. On approve we are editing source_post_moderation entry and then trigger the CDC (post will be published a bit later). 
Do I have to pass the SourcePostModeration item here?

Reply:
It can be partial that's fine I think.
Just pass which ever we have at that time.

CC: @idoshamun

Reply:
yap. whatever you have at this point is fine

Reply:
@rebelchris updated `onApprove` event using `postLogEvent`. 

Kept the old version for Rejection as we don't have any data in return for this action from API, let me know if you think that I have to change it for handle this case 👀

Reply:
But you have it locally right the rejection data that you send/interact with?

Reply:
Yes, function right now receive postId, I can use it (or make function accept more data) and move the log into a `.then()` function after mutation. Do you think this should be ok?

Reply:
Sorry @rebelchris , was wrong and after double check I've the same data even for rejection. Updated with a common function 👍